### PR TITLE
lag_interfaces: Handled change mode stdout error

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

Depends-On: https://github.com/ansible-collections/ansible.netcommon/pull/115
##### SUMMARY
related to  #95 

Handled the terminal output error **% Cannot change mode; remove all members and try again.**

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
terminal/eos.py

### CODE REVIEW

File: plugins/terminal/eos.py [hunk 1]
```diff
@@ -52,6 +52,9 @@ class TerminalModule(TerminalBase):
         re.compile(br"% Subnet [0-9a-f.:/]+ overlaps", re.I),
         re.compile(br"Maximum number of pending sessions has been reached"),
         re.compile(br"% Prefix length must be less than"),
+        re.compile(
+            br"% Cannot change mode; remove all members and try again.", re.I
```
Comment (Qalthos) [inline plugins/terminal/eos.py:56]: Is `re.I` necessary here?

Also, can you please add a comment noting what sort of command causes this response?

Comment (GomathiselviS) [reply plugins/terminal/eos.py:56]: I added re.I as a precaution, if there is a change in the case in future. But now that I think about it, the chance for such change is slim.

**Comment by pabelanger** [comment]
recheck

File: plugins/terminal/eos.py [hunk 2]
```diff
@@ -52,6 +52,10 @@ class TerminalModule(TerminalBase):
         re.compile(br"% Subnet [0-9a-f.:/]+ overlaps", re.I),
         re.compile(br"Maximum number of pending sessions has been reached"),
         re.compile(br"% Prefix length must be less than"),
+        # terminal error when a configured port-channel's mode is modified.
```
Comment (Qalthos) [inline plugins/terminal/eos.py:55]: Can you give a more concrete example of the actual command?

File: plugins/terminal/eos.py [hunk 3]
```diff
@@ -52,6 +52,17 @@ class TerminalModule(TerminalBase):
...
         re.compile(br"Maximum number of pending sessions has been reached"),
         re.compile(br"% Prefix length must be less than"),
+        # terminal error when a configured channel-group's mode is modified.
+        # 
+        # veos(config-if-Et2)#show active 
+        # interface Ethernet2
+        # channel-group 15 mode on
+        # -----------------------------
+        # veos(config-if-Et2)#channel-group 15 mode active
+        # % Cannot change mode; remove all members and try again.
```
Comment (Qalthos) [inline plugins/terminal/eos.py:62]:
```suggestion
        # May be returned in response to `channel-group <number> mode <mode>`
```
This is all I'm really after. I just want stderr entries to have some indication of what causes them so we can have a chance to ever audit them some day down the road.

**Pull request merged**